src/components/IconLibrary/IconLibrary.module.scss

```diff
@@ -0,0 +1,199 @@
+.icon-page {
+  margin-left: -$spacing-05;
+  margin-right: -$spacing-05;
+
+  // prevent jank when icons load
+  min-height: 60vh;
+}
+
+.filter-row {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0 $spacing-05;
+  @include carbon--breakpoint('md') {
+    align-items: flex-end;
+    padding: 0;
+    flex-direction: row;
+  }
+}
+
+.filter-row :global(.bx--search) {
+  @include carbon--breakpoint('md') {
+    width: 60%;
+    margin-right: $spacing-11;
+  }
+}
+
+.filter-row :global(.bx--search .bx--search-input) {
+  height: 40px;
+}
+
+.filter-row :global(.bx--dropdown__wrapper) {
+  flex-grow: 1;
+  width: 100%;
+  margin-top: $spacing-05;
+  @include carbon--breakpoint('md') {
+    margin-top: 0;
+    width: auto;
+  }
+}
+
+.icon-library {
+  overflow: hidden;
+}
+
+.icon-category .category-title {
+  margin: 0;
+  margin-left: $spacing-05;
+  margin-top: $spacing-11;
+}
+
+.icon-category .subcategory-title {
+  margin: 0;
+  margin-top: $spacing-09;
+  margin-left: $spacing-05;
+}
+
+// i.e. fallback
+.icon-grid {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.icon-grid {
+  margin-top: $spacing-05;
+  display: grid;
```

Comment:
Awesome to see CSS Grid, is this okay for browser support? If so then 💯

Reply:
I have a fallback right above.